### ISSUE TITLE
Knapsack may not be compatible with Cucumber 1.2.5

### ISSUE THREAD

**Issue by johngluckmdsol** [KnapsackPro/knapsack_pro-ruby#5]
I actually got this error with Knapsack as well.  I'm hoping you know what it means and can help me resolve it.   

undefined method `scenario_outline' for #<Cucumber::Ast::Scenario:0x000000073044e8> (NoMethodError)
 /home/travis/build/<path removed>/vendor/bundle/ruby/2.2.0/gems/knapsack_pro-0.6.1/lib/knapsack_pro/adapters/cucumber_adapter.rb:15:in `Around'

**Comment by ArturT** [comment]
What is your cucumber gem version?

**Comment by ArturT** [comment]
@johngluckmdsol  I can't reproduce the problem because I'm not able to install cucumber 1.5:

`Could not find gem 'cucumber (~> 1.5)' in any of the gem sources listed in your Gemfile
or available on this machine.`

There is no cucumebr version 1.5 on rubygems:
https://rubygems.org/gems/cucumber/versions

Could you update your cucumber gem instead?

**Comment by johngluck65** [comment]
My understanding is, unfortunately, that our cucumber is painful to upgrade so I was hoping to avoid that exercise, but this confirms my conclusion that the only to get to troubleshoot this issue it to upgrade cucumber.  Thanks.  You can close this for now

**Comment by ArturT** [comment]
If you could share with me your cucumber 1.5 version (you can publish it on github) then I can try to fix the bug. Was 1.5 release an official one or is it your fork?

**Comment by johngluckmdsol** [comment]
Ah, okay.  I'm sorry, I got confused.  This is actually cucumber 1.2.5. I'll change the title

**Comment by ArturT** [comment]
I did fix. Could you try if this works for you? If yes then I'm going to release a new gem version for both gems knapsack and knapsack_pro.

Please add in your Gemfile:
```ruby
gem 'knapsack_pro', github: 'KnapsackPro/knapsack_pro-ruby', branch: 'support-old-cucumber'
```

**Comment by johngluckmdsol** [comment]
It appears to be working.  Thanks for the speedy fix!!!

**Comment by ArturT** [comment]
I've just released knapsack_pro 0.7.0. Please use it from rubygems. I'm going to remove branch support-old-cucumber soon.

Thanks for reporting issue!  

If you will need any help or more API keys then you can find my email here http://knapsackpro.com/
